tutorials/3d/portals/tutorial_simple.rst

```diff
@@ -0,0 +1,83 @@
+- I've placed these boxes as children of a Spatial I have called 'Freeform', just to keep our scene tree neat and tidy.
+- I've also used a green material to make the boxes stand out more.
+- Let's also place an :ref:`OmniLight<class_OmniLight>` so it will be autoplaced in one of the rooms.
+
+Step 5
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple5.png
+
+- Next comes a crucial stage, we must let the RoomManager know where the rooms are!
+- Select the RoomManager and look in the Inspector window in the ``Paths`` section.
+- You need to assign the ``RoomList`` to point to the 'RoomList' node we created earlier (which is the parent of all the rooms).
+
+Step 6
+~~~~~~
+.. image:: tutorial_simple/img/tutorial_simple6.png
+
+- Make sure you have saved your project before this next step (it is always a good idea to save and make a backup before converting).
+- Select the RoomManager, and you will see a button in the toolbar at the top of the 3d view called ``Convert Rooms``. Press this button.
+- If all goes well, the RoomManager will have created the runtime data (the ``room graph``) to do culling at runtime.
+- You can see a log of the conversion process in the output window. This is helpful for finding problems.
+- If you now move the editor camera inside the rooms, you should see the meshes in the opposite room being culled depending on what you can see through the portal.
+
+Conclusion
+~~~~~~~~~~
+This concludes this simple tutorial. Don't be afraid to experiment with the new room system you have created.
+
+Some things to try:
+^^^^^^^^^^^^^^^^^^^
+- Create different types of geometry - CSG nodes, Particle systems, Multimeshes
+- Try creating a Camera and adding it to the scene. If you run the scene you will notice that the portal culling is not active. This is because the ``room graph`` must be created each time you load a level, by converting the rooms. Instead of using a button in the editor, in real games you call a function in the RoomManager to convert the level, called ``rooms_convert()``. Try this out with a script, perhaps running within a ``_ready`` function.
+- The geometry you created so far is all ``STATIC`` (non-moving). If you look in the inspector for geometry nodes, you will see they derive from ``CullInstance``. Here you can set the ``portal_mode`` for objects in the portal system. This determines how the node is processed.
+- If you now write a script to move one of your objects within a room and view it through a Camera as the scene runs, you may notice that the object gets culled incorrectly. This is because ``STATIC`` objects are assumed not to move in the system. If you instead change the object to ``DYNAMIC``, it should now update the culling correctly.
+- There are several ``portal_modes``, these are described in the main documentation.
+- Try turning the portal on and off at runtime from your script! You can call ``set_portal_active`` to open and close the portal. 
```

Comment:
```suggestion
Some things to try
^^^^^^^^^^^^^^^^^^

- Create different types of geometry. CSG nodes, particle systems and MultiMeshes are all supported by the portal system.
- Try creating a Camera and adding it to the scene. If you run the scene, you will notice that the portal culling is not active. This is because the ``room graph`` must be created each time you load a level, by converting the rooms. Instead of using a button in the editor, in real games you call a function in the RoomManager to convert the level, called ``rooms_convert()``. Try this out with a script, perhaps running within a ``_ready()`` function.
- The geometry you created so far is all ``STATIC`` (non-moving). If you look in the inspector for geometry nodes, you will see they derive from ``CullInstance``. Here you can set the **Portal Mode** for objects in the portal system. This determines how the node is processed.
- If you now write a script to move one of your objects within a room and view it through a Camera as the scene runs, you may notice that the object gets culled incorrectly. This is because ``STATIC`` objects are assumed not to move in the system. If you instead change the object to ``DYNAMIC``, it should now update the culling correctly.
- There are several ``portal_modes``, these are described in the main documentation and the class reference.
- Try turning the portal on and off at runtime from your script. You can call ``set_portal_active()`` to open and close the portal. This can be used for things such as doors that open and close.
```